Kotlin OpenAPI generated project fail with latest 1.6.13 or 2.0.9 codegen-cli

I tried to generate a REST project using Kotlin and OpenAPI, but the project does not build or run properly since the Kotlin template is still using version 1.6.0 of the light4j libraries. The old libraries are missing some fields in the `ValidatorConfig` class. In addition the 1.6.13 generated project is missing the `val jacksonDatabindVersion` definition in `build.gradle.kts` so Gradle does not run at all.

Here is the stacktrace I get on `./gradlew clean build run` with 2.0.9:
```
Exception in thread "main" java.lang.ExceptionInInitializerError
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:315)
        at com.networknt.handler.Handler.splitClassAndName(Handler.java:499)
        at com.networknt.handler.Handler.initStringDefinedHandler(Handler.java:409)
        at com.networknt.handler.Handler.initHandlers(Handler.java:89)
        at com.networknt.handler.Handler.init(Handler.java:70)
        at com.networknt.server.Server.start(Server.java:154)
        at com.networknt.server.Server.init(Server.java:126)
        at com.networknt.server.Server.main(Server.java:111)
Caused by: java.lang.RuntimeException: Unable to load openapi-validator.yml as object.
        at com.networknt.config.Config$FileConfigImpl.loadSpecificConfigFileAsObject(Config.java:323)
        at com.networknt.config.Config$FileConfigImpl.loadObjectConfig(Config.java:331)
        at com.networknt.config.Config$FileConfigImpl.getJsonObjectConfig(Config.java:232)
        at com.networknt.config.Config$FileConfigImpl.getJsonObjectConfig(Config.java:242)
        at com.networknt.openapi.ValidatorHandler.<clinit>(ValidatorHandler.java:52)
        ... 9 more
Caused by: java.lang.IllegalArgumentException: Unrecognized field "validateResponse" (class com.networknt.openapi.ValidatorConfig), not marked as ignorable (3 known properties: "enabled", "skipBodyValidation", "logError"])
 at [Source: UNKNOWN; line: -1, column: -1] (through reference chain: com.networknt.openapi.ValidatorConfig["validateResponse"])
        at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:3922)
        at com.fasterxml.jackson.databind.ObjectMapper.convertValue(ObjectMapper.java:3853)
        at com.networknt.config.CentralizedManagement.convertMapToObj(CentralizedManagement.java:83)
        at com.networknt.config.CentralizedManagement.mergeObject(CentralizedManagement.java:44)
        at com.networknt.config.Config$FileConfigImpl.loadSpecificConfigFileAsObject(Config.java:318)
        ... 13 more
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "validateResponse" (class com.networknt.openapi.ValidatorConfig), not marked as ignorable (3 known properties: "enabled", "skipBodyValidation", "logError"])
 at [Source: UNKNOWN; line: -1, column: -1] (through reference chain: com.networknt.openapi.ValidatorConfig["validateResponse"])
        at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
        at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:840)
        at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:1179)
        at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1592)
        at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1570)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:294)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
        at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:3917)
        ... 17 more
```

--- a/light-rest-4j/src/main/resources/templates/restkotlin/gradleProperties.rocker.raw
+++ b/light-rest-4j/src/main/resources/templates/restkotlin/gradleProperties.rocker.raw
@@ -2,7 +2,7 @@
 @args (Any config)
 # Versions of Frequently used Libraries
 kafkaVersion=2.0.0
-light4jVersion=1.6.13
+light4jVersion=1.6.14-SNAPSHOT
 jacksonVersion=2.10.0
 jacksonDatabindVersion=2.10.0
 undertowVersion=2.0.28.Final

